common/script/content/eggs.js

```diff
@@ -0,0 +1,262 @@
+import {
+  assign,
+  defaults,
+  each,
+} from  'lodash';
+import t from './translation';
+
+function applyEggDefaults (set, config) {
+  each(set, (egg, key) => {
+    defaults(egg, {
+      canBuy: config.canBuy,
+      value: 3,
+      key,
+      notes: t('eggNotes', {
+        eggText: egg.text,
+        eggAdjective: egg.adjective,
+      }),
+      mountText: egg.text,
+    });
+  });
+}
+
+function hasQuestAchievementFunction (key) {
+  return (user) => {
+    return user.achievements.quests &&
+      user.achievements.quests[key] > 0;
+  };
+}
+
+let drops = {
+  Wolf: {
```

Comment:
It's something we _should_ have addressed in the v2 -> v3 switch, but ultimately, I don't think that it is that big of a deal.